src/Client.js

```diff
@@ -51,6 +51,8 @@ export default class Client {
   users: Users;
   webhooks: Webhooks;
 
+  analyticsCallback: ?AnalyticsCallback;
```

Comment:
> then we have to use dependency injection around the redux store, and the injection is causing bad require loops

@rickharris gotcha, yea that is annoying. In that case, can we keep this property private by prefixing it with `_`?

```suggestion
  _analyticsCallback: ?AnalyticsCallback;
```

Reply:
Yeah, sure, if that's the pattern we want to keep. I wasn't really planning on documenting this or anything 😆. The `_` doesn't do anything special right? People will still see it if they inspect the client object?

Reply:
Yup, just for the sdk as one of it's features is a public interface. `_` helps signal this is not documented and this will likely change without notice